isos/bootstrap/bootstrap

```diff
@@ -32,15 +32,9 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
         mkdir -p ${MOUNTPOINT}/dev ${MOUNTPOINT}/proc ${MOUNTPOINT}/sys ${MOUNTPOINT}/etc
         # ln -sf /proc/mounts ${MOUNTPOINT}/etc/mtab
 
-        # ensure both the source and destination files exist for bind mounting
-        touch ${MOUNTPOINT}/etc/hostname ${MOUNTPOINT}/.tether-init/hostname
-        touch ${MOUNTPOINT}/.tether-init/resolv.conf ${MOUNTPOINT}/.tether-init/resolv.conf
+        touch ${MOUNTPOINT}/etc/hostname
         touch ${MOUNTPOINT}/etc/hosts
-        echo "127.0.0.1 localhost localhost.localdomain" > ${MOUNTPOINT}/.tether-init/hosts
```

Comment:
The loopback entries are now done in BaseOperations.Setup as per @caglar10ur suggestion.